internal/pkg/addons/testdata/template.yml

```diff
@@ -1,19 +1,19 @@
 Parameters:
-  Project:
+  App:
     Type: String
```

Comment:
Is this test file used anywhere?

Reply:
Also I wonder if we should first do the translation of "app -> svc" and then tackle "project -> app" to reduce the scope of the changes.

Reply:
in output_test I think

Reply:
oh i was confused because i didn't see any changes to `Project` in the PR, yet the tests pass 🤔

Reply:
ignore thi

> Also I wonder if we should first do the translation of "app -> svc" and then tackle "project -> app" to reduce the scope of the changes.

ignore this comment, i think what's here is nice. It's okay if the CLI itself doesn't work while we are on this branch but the unit tests must pass for each PR 👍